src/ResourceManager/Kubernetes/Commands.Kubernetes/Commands/StopAzureRmKubernetesDashboard.cs

```diff
@@ -0,0 +1,51 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Kubernetes
+{
+    [Cmdlet("Stop", KubeNounStr + "Dashboard")]
+    public class StopDashboard : KubeCmdletBase
```

Comment:
I don't know that we need this.  If we return a job from the first cmdlet, the user can simply stop it.

Reply:
It not needed by someone that understands powershell jobs, but it does give a simple way to kill the job if you are not adept with Jobs.